### DISCUSSION TITLE
Multiple applications on same Cluster

### DISCUSSION THREAD

**Discussion by adaosantos** [aws/copilot-cli#4608] [Q&A]
In my case I have one cluster to provide same services, example:

```
-  Cluster 1
 -- Service 1
 -- Service 2
 -- Service 3
```

And each service is in a different github repository, today I use terraform to provisioning that infra:

```
- Cluster Production 1
 -- Service 1
 -- Service 2
 -- Service 3
 
- Cluster QA 1 
 -- Service 1
 -- Service 2
 -- Service 3 
```

How I can reproduce this on copilot manifests ?

**Comment by efekarakus** [comment]
Hi @adaosantos !

With Copilot, by default ECS Services will share the Cluster within an environment 👍 so I believe your usecase should be a direct fit for copilot.

For example,
```console
$ copilot env init --name production-1
$ copilot env init --name qa-1
$ copilot svc deploy --name service-1 --env production-1
$ copilot svc deploy --name service-2 --env production-1
```

You shouldn't need to do any configuration with manifest files to get the architecture described above.
Does this help?
